src/serverdlgbase.ui

```diff
@@ -219,7 +219,7 @@
          <item>
           <widget class="QLabel" name="lblLanguage">
            <property name="text">
-            <string>Language</string>
+            <string extracomment="Consider add &quot;(Lang)&quot; as a suffix to ensure that users who selected the wrong language find the correct button">Language</string>
```

Comment:
Should be documented somewhere else (for the next release): Translators should add this in their language.

Reply:
"suffix" is a bit technical perhaps.

```suggestion
            <string extracomment="Consider adding &quot;(Lang)&quot; at the end so that users who select the wrong language can find the correct button">Language</string>
```